doc/conf.py

```diff
@@ -8,7 +8,7 @@
 
 # Generate indices for the Python modules in the folder above
 subprocess.call(
-    'sphinx-apidoc -f -M -o api ..',
+    'sphinx-apidoc -f -M -o api ../tensorwaves ../tensorwaves/*constraint*',
```

Comment:
Agree, but why the `../tensorwaves/*constraint*`?

Reply:
Because thats actually a third party module that i copied in (instead of using pip) because I'm not sure about the future. Since this module has a lot of doc string problems I do not want any docstrings generated for this and this statement excludes it